src/SprykerEco/Shared/AmazonPay/AmazonPayConfig.php

```diff
@@ -224,4 +224,20 @@ public function getEnableIsolateLevelRead()
     {
         return (bool)$this->get(AmazonPayConstants::ENABLE_ISOLATE_LEVEL_READ, false);
     }
+
+    /**
+     * @return string
+     */
+    public function getSuccessPaymentUrl(): string
+    {
+        return $this->get(AmazonPayConstants::SUCCESS_PAYMENT_URL);
```

Comment:
Not really sure if amazon url should be configured.

Reply:
Its URL where the user will be redirected in Yves in case MFA challenge is passed.